packages/@o3r/core/schematics/store/common/mocks/example-module.ts

```diff
@@ -1,26 +1,38 @@
-export const commonModuleContent = `import { InjectionToken, ModuleWithProviders, NgModule } from '@angular/core';
+export const commonModuleContent = `import { InjectionToken, ModuleWithProviders, NgModule, Optional } from '@angular/core';
 import { Action, ActionReducer, StoreModule } from '@ngrx/store';
 
 import { EffectsModule } from '@ngrx/effects';
 import { ExampleEffect } from './example.effect';
 import { exampleReducer } from './example.reducer';
 import { EXAMPLE_STORE_NAME, ExampleState } from './example.state';
 
+
+/** Action Reducer map for Example store */
+type ExampleActionReducer<T extends ExampleState=ExampleState, V extends Action=Action> = ActionReducer<T, V>;
+
+/** Internal Token of the Example reducer */
+const INTERNAL_EXAMPLE_REDUCER_TOKEN = new InjectionToken<ActionReducer<ExampleState, Action>>('Internal Example Reducer');
```

Comment:
Why not using `ExampleActionReducer`

```suggestion
const INTERNAL_EXAMPLE_REDUCER_TOKEN = new InjectionToken<ExampleActionReducer>('Internal Example Reducer');
```